.travis.yml

```diff
@@ -8,10 +8,21 @@ os:
   - osx
   - windows
 cache: cargo
+before_install:
+  - |
+    mkdir glslang
+    cd glslang
+    wget https://github.com/KhronosGroup/glslang/releases/download/master-tot/glslang-master-linux-Release.zip
+    unzip -u -q glslang-master-linux-Release.zip
+    cd ..
+    export PATH=${TRAVIS_BUILD_DIR}/glslang/bin:$PATH
+    glslangValidator --version
+
 matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
-script:
+  script:
```

Comment:
It looks like you indented this accidentally, so Travis is trying to execute the default build script with not features enabled.

Reply:
Oh, whoops. Thank you for catching that!